### PR TITLE
v3.3/glfw: expose native wayland methods

### PR DESCRIPTION
**Pull request by rajveermalviya**
exposes native wayland & EGL methods.

https://github.com/glfw/glfw/blob/df8d7bc892937a8b0f7c604c92a9f64f383cf48c/include/GLFW/glfw3native.h#L420

### CODE REVIEW

**Comment by rajveermalviya** [comment]
@dmitshur PTAL

File: v3.3/glfw/native_linbsd_wayland.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// +build linux,wayland freebsd,wayland netbsd,wayland
```
Comment (Jacalz) [inline v3.3/glfw/native_linbsd_wayland.go]: Shouldn't this contain `openbsd,wayland` as well?

Comment (rajveermalviya) [reply v3.3/glfw/native_linbsd_wayland.go]: yes, updated

Comment (Jacalz) [reply v3.3/glfw/native_linbsd_wayland.go]: Thanks.

Comment (dmitshur) [reply v3.3/glfw/native_linbsd_wayland.go]: Now that the new file has `openbsd,wayland` added, it seems `native_linbsd_x11.go` should also be updated so its build constraint is `openbsd,!wayland`.

Comment (Jacalz) [reply v3.3/glfw/native_linbsd_wayland.go]: Good catch. That must have been missed when the OpenBSD support was added in.

Comment (rajveermalviya) [reply v3.3/glfw/native_linbsd_wayland.go]: > Now that the new file has openbsd,wayland added, it seems native_linbsd_x11.go should also be updated so its build constraint is openbsd,!wayland.

done.

**Review by Jacalz — APPROVED** [review]
Sorry for the late review. Looks good to me.

**Comment by dmitshur** [comment]
Thanks.

**Pull request merged**